src/main/java/com/google/sps/servlets/YoutubeApi.java

```diff
@@ -46,14 +56,32 @@ private static YouTube getService() throws GeneralSecurityException, IOException
 
   public static List<YoutubePost> getYoutubePost() throws YoutubeApiException {
     List<YoutubePost> list = new ArrayList();
+    List<CommentData> commentList = new ArrayList();
     String id = "Ks-_Mh1QhMc";
     try {
       VideoListResponse response =
           getService().videos().list("snippet").setKey(DEVELOPER_KEY).setId(id).execute();
       Video video = response.getItems().get(0);
-      YoutubePost newPost = new YoutubePost(
-          video.getSnippet().getTitle(), video.getSnippet().getDescription(), video.getId());
+      /**Retrieving comments before creating new post */
```

Comment:
nit: javadoc style comment (using /** */) shouldn't be used inside functions. Use normal comments ( // ) instead.